financial-connections/src/main/java/com/stripe/android/financialconnections/exception/FinancialConnectionsError.kt

```diff
@@ -15,5 +16,24 @@ internal abstract class FinancialConnectionsError(
     stripeException.cause,
     stripeException.message
 ) {
+
     override fun analyticsValue(): String = "fcError"
+
+    constructor(
+        name: String,
+        stripeError: StripeError? = null,
+        requestId: String? = null,
+        statusCode: Int = 0,
+        cause: Throwable? = null,
+        message: String? = stripeError?.message
+    ) : this(
+        name = name,
+        stripeException = object : StripeException(
+            stripeError = stripeError,
+            requestId = requestId,
+            statusCode = statusCode,
+            cause = cause,
+            message = message
+        ) {}
+    )
```

Comment:
just a helper to facilitate creating a `FinancialConnectionsError` without a StripeException.